src/libraries/Directory.Build.props

```diff
@@ -300,6 +300,7 @@
       We do this manually and use the Configuration instead.
     -->
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+    <TargetFramework Condition="'$(MSBuildProjectExtension)' == '.proj'">$(BuildTargetFramework)</TargetFramework>
```

Comment:
This will include a TargetFramework for, build.proj, and others, right?

Reply:
yes it will work for build.proj, tests.proj, bin-native.proj, helix.proj

Reply:
I mean, do we need it for other projects? That could cause weird stuff, like sendtohelix.proj and build.proj shouldn't have a TargetFramework.

Reply:
we create intermediateOutputPaths for helix.proj, currently they are broken because TargetFramework is not set.

Reply:
Why do we create IntermediateOutputPaths for sendtohelix.proj? I don't get that, we never run it as part of the build, we just call it with /t:test in helix.yml. 

Also, build.proj is the orchestrator of the vertical build, should that have a TargetFramework?

Reply:
Right, I'm concerned about this as well. Do we actually want to force every project to have TargetFramework(s) set? I don't think that's the right path forward.